src/Resources/contao/dca/tl_user.php

```diff
@@ -346,6 +346,18 @@
 			'eval'                    => array('multiple'=>true),
 			'sql'                     => "blob NULL"
 		),
+		'imageSizes' => array
+		(
+			'label'                   => &$GLOBALS['TL_LANG']['tl_user']['imageSizes'],
+			'exclude'                 => true,
+			'inputType'               => 'checkbox',
+			'options_callback'        => function () {
+				return System::getContainer()->get('contao.image.image_sizes')->getAllOptions();
```

Comment:
No it should not. Because the service is not supposed to only return permissions. @ausi might use it for his PR on the image services, that's why the name is pretty specific.